<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplaceDlg</class>
 <widget class="QWidget" name="FindAndReplaceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FindAndReplaceDlg{
	background-image:url(:/findAndReplace/images/findAndReplace/rectangle.svg)
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="LineInput_findWord">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>11</y>
        <width>271</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-right: 90;
background-image: url(:/findAndReplace/images/findAndReplace/inputBox.svg);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineInput_replace">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>47</y>
        <width>271</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-right: 30;
background-image: url(:/findAndReplace/images/findAndReplace/inputBox.svg);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_REG">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>8</y>
        <width>30</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>使用正则表达式</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: none;
	image: url(:/findAndReplace/images/findAndReplace/REG-unselected.svg);
}
QPushButton::hover{	
border: none;
	image: url(:/findAndReplace/images/findAndReplace/REG-selected.svg);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_findWholeWord">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>8</y>
        <width>30</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>全字匹配</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: none;
	image: url(:/findAndReplace/images/findAndReplace/wholeWord-unselected.svg);
}
QPushButton::hover{	
border: none;
	image: url(:/findAndReplace/images/findAndReplace/wholeWord-selected.svg);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_findUpOrLow">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>8</y>
        <width>30</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>区分大小写</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: none;
image: url(:/findAndReplace/images/findAndReplace/inputOnlyUpper-unselected.svg);
}
QPushButton::hover{	
border: none;
image: url(:/findAndReplace/images/findAndReplace/inputOnlyUpper-selected.svg.svg);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_replaceUpOnly">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>45</y>
        <width>30</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>保留大小写</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: none;
	image: url(:/findAndReplace/images/findAndReplace/replaceOnlyUpper-unselected.svg.svg);
}
QPushButton::hover{	
border: none;
	image: url(:/findAndReplace/images/findAndReplace/replaceOnlyUpper-selected.svg);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" rowminimumheight="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="Btn_replaceOne">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>替换</string>
           </property>
           <property name="styleSheet">
            <string notr="true">/**正常情况下样式**/
QPushButton{
image: url(:/findAndReplace/images/findAndReplace/replaceOne-unselected.svg);
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
	image: url(:/findAndReplace/images/findAndReplace/replaceOne-selected.svg);
}

/**鼠标按压下去的样式**/
QPushButton::checked{
	image: url(:/findAndReplace/images/findAndReplace/replaceOne-selected.svg);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
image: url(:/findAndReplace/images/findAndReplace/replaceOne-unselected.svg);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_replaceAll">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>全部替换</string>
           </property>
           <property name="styleSheet">
            <string notr="true">/**正常情况下样式**/
QPushButton{
image: url(:/findAndReplace/images/findAndReplace/replaceAll-unselected.svg);
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
	image: url(:/findAndReplace/images/findAndReplace/replaceAll-selected.svg);
}

/**鼠标按压下去的样式**/
QPushButton::checked{
	image: url(:/findAndReplace/images/findAndReplace/replaceAll-selected.svg);
}

/**按钮失能情况下样式**/
QPushButton::disabled{
image: url(:/findAndReplace/images/findAndReplace/replaceAll-unselected.svg);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="Btn_close">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>关闭</string>
        </property>
        <property name="styleSheet">
         <string notr="true">

/**正常情况下样式**/
QPushButton{
image: url(:/findAndReplace/images/findAndReplace/close-unselected.svg);
padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
	image: url(:/findAndReplace/images/findAndReplace/close-selected.svg);
padding: 5px;
}

/**鼠标按压下去的样式**/
QPushButton::checked{
	image: url(:/findAndReplace/images/findAndReplace/close-selected.svg);
padding: 5px;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
image: url(:/findAndReplace/images/findAndReplace/close-unselected.svg);
padding: 5px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Btn_next_word">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>查找下一个</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
/**正常情况下样式**/
QPushButton{
image: url(:/findAndReplace/images/findAndReplace/next-unselected.svg);
padding:5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
	image: url(:/findAndReplace/images/findAndReplace/next-selected.svg);
padding: 5px;
}

/**鼠标按压下去的样式**/
QPushButton::checked{
	image: url(:/findAndReplace/images/findAndReplace/next-selected.svg);
padding: 5px;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
image: url(:/findAndReplace/images/findAndReplace/next-unselected.svg);
padding:5px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="findingResult">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Btn_pre_word">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>查找上一个</string>
        </property>
        <property name="styleSheet">
         <string notr="true">

/**正常情况下样式**/
QPushButton{
image: url(:/findAndReplace/images/findAndReplace/previous-unselected.svg);
padding:5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
	image: url(:/findAndReplace/images/findAndReplace/previous-selected.svg);
padding: 5px;
}

/**鼠标按压下去的样式**/
QPushButton::checked{
	image: url(:/findAndReplace/images/findAndReplace/previous-selected.svg);
padding: 5px;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
image: url(:/findAndReplace/images/findAndReplace/previous-unselected.svg);
padding:5px;
}X</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
